src/pages/editor/curators/AddCuratorDialog.tsx

```diff
@@ -0,0 +1,183 @@
+    retry: false,
+    cacheTime: 0, // Disable caching since this user in many cases will be changed, and the cached data then will be outdated
+  });
+
+  useEffect(() => {
+    const currentUser = userQuery.data;
+    if (currentUser) {
+      let viewingScope = currentUser.viewingScope.includedUnits;
+
+      const newOrganizationId = !viewingScope.includes(currentOrganization.id) ? currentOrganization.id : null;
+      if (newOrganizationId) {
+        // Remove organizations conflicting with the new organization
+        const childOrganizationIds = getAllChildOrganizations(currentOrganization.hasPart).map((unit) => unit.id);
+        viewingScope = viewingScope.filter(
+          (id) => !parentOrganizationIds.includes(id) && !childOrganizationIds.includes(id)
+        );
+
+        viewingScope.push(newOrganizationId);
+      }
+
+      const initialValues: InstitutionUser = {
+        ...currentUser,
+        viewingScope: {
+          ...currentUser.viewingScope,
+          includedUnits: viewingScope,
+        },
+      };
+
+      setUserInitialValues(initialValues);
+    }
+  }, [currentOrganization, parentOrganizationIds, userQuery.data]);
+
+  return (
+    <Dialog open={open} onClose={closeDialog} maxWidth="sm" fullWidth>
+      <DialogTitle>{t('editor.curators.add_curator')}</DialogTitle>
+
+      <DialogContent>
+        <Autocomplete
+          options={employeeSearchQuery.data?.hits ?? []}
+          renderOption={(props, option) => (
+            <li {...props} key={option.id}>
+              {getFullCristinName(option.names)}
+            </li>
+          )}
+          onInputChange={(_, value, reason) => {
+            if (reason === 'clear' || reason === 'reset') {
+              setSearchQuery('');
+              setSelectedPerson(null);
+              setUserInitialValues(null);
+            } else {
+              setSearchQuery(value);
+            }
+          }}
+          onChange={async (_, value) => {
+            setSearchQuery('');
+            setSelectedPerson(value);
+          }}
+          getOptionLabel={(option) => getFullCristinName(option.names)}
+          isOptionEqualToValue={(option, value) => option.id === value.id}
+          loading={employeeSearchQuery.isFetching}
+          sx={{ mb: '1rem' }}
+          renderInput={(params) => (
+            <AutocompleteTextField
+              data-testid={dataTestId.editor.curatorsSearchForPersonField}
+              {...params}
+              label={t('common.person')}
+              isLoading={employeeSearchQuery.isFetching}
+              placeholder={t('common.search')}
+              showSearchIcon
+            />
+          )}
+        />
+
+        {selectedPerson && (
+          <>
+            {userQuery.isLoading || (userQuery.data && !userInitialValues) ? (
+              <CircularProgress aria-label={t('editor.curators.add_curator')} />
```

Comment:
Ikke skeleton?

Reply:
Mye mer jobb å få inn et skeleton som passer, enn å bare legge inn en spinner, så tok the easy way 😅 
Vi burde kanskje snart ha den praten med designer om å få en strategi på hvordan vi skal vise loading, for nå er det hipp som happ :/